specs/_features/eip7594/das-core.md

```diff
@@ -235,11 +235,11 @@ Due to the deterministic custody functions, a node knows exactly what a peer sho
 
 ## Reconstruction and cross-seeding
 
-If the node obtains 50%+ of all the columns, they can reconstruct the full data matrix via `recover_matrix` helper.
+If a node fails to get a column on the column subnet, a node can also utilize the Req/Resp message to query the missing column from other peers.
```

Comment:
This part feels like it belongs in some other section, it doesn't seem specific to reconstruction

Reply:
This paragraph was already in this subsection, just one sentence later. I agree we should move it somehere else. I will try to find a better place for it and move it.